syntax = "proto3";

package hello;

service PaymentService {
    rpc paymentLogic (PaymentRequest) returns (PaymentResponse);
}

message PaymentRequest {
    string orderId = 1;
}

message PaymentResponse {
    string verdict = 1;
}